react/features/base/react/components/native/Container.js

```diff
@@ -50,19 +59,24 @@ export default class Container<P: Props> extends AbstractContainer<P> {
 
         // onClick & touchFeedback
         if (element && onClickOrTouchFeedback) {
+            const touchableProps = {
+                accessibilityLabel,
+                accessible,
+                onLongPress,
+                onPress: onClick
+            };
+
             element
-                = React.createElement(
-                    touchFeedback
-                        ? TouchableHighlight
-                        : TouchableWithoutFeedback,
-                    {
-                        accessibilityLabel,
-                        accessible,
-                        onLongPress,
-                        onPress: onClick,
-                        ...touchFeedback && { underlayColor }
-                    },
-                    element);
+                = touchFeedback
```

Comment:
Why is this change necessary? I find it harder to read than the previous implementation.

Reply:
Because we were feeding an invalid prop to the component when ```touchFeedback``` was false